### ISSUE TITLE
Neo block not returning any data

### ISSUE THREAD

**Issue by angeloatzicht** [spicywebau/craft-neo#516]
#### Description
In our CMS we have added a Neo block with nested items. In our case it is a FAQ. 
The structure is shown below.

- Neo
- - Description (text)
- - Q&A
- - - Question
- - - Answer

It has been working for a long time, but (not sure when it broke down) last week we saw that it is not working anymore.
In the CMS we still see everything working (visually), we can still edit, delete and store any changes made but the data is never returned in the GraphQL queries.

We tried requesting the data through the GraphiQL but it always returns an empty array. An example of our GraphQL query below:

```
{
  entries(slug: "my_slug", site: "*") {
    title
    ... on something_something_Entry {
      neo {
        ... on neo_questionanswers_BlockType {
          children {
            ... on neo_questionanswersentry_BlockType {
              question
              answer
            }
          }
        }
      }
    }
  }
}
```

#### Steps to reproduce
Add a new Neo block like the structure above

#### Other information
- Neo version: latest (also tried downgrading to 2.11.14)
- Craft version: 3.6.4
- Is eager-loading used? no

**Comment by ttempleton** [comment]
Thanks for reporting this! Just added a fix which will be included in the next Neo release.

**Comment by we-do** [comment]
I see that this was addressed and the fix was pushed to master. Can you bump the version because composer is still on 2.11.17 from 5 days ago and Craft can't see the update.

**Comment by ttempleton** [comment]
@we-do There are a couple of other issues to be closed first, but aiming to have Neo 2.11.18 ready for release later today.

**Comment by ttempleton** [comment]
Just released 2.11.18 including the fix for this.
